openpype/hosts/maya/plugins/publish/validate_rendersettings.py

```diff
@@ -242,6 +242,14 @@ def get_invalid(cls, instance):
             instance.context.data["project_settings"]["maya"]["publish"]["ValidateRenderSettings"].get(  # noqa: E501
                 "{}_render_attributes".format(renderer)) or []
         )
+        settings_lights_flag = instance.context.data["project_settings"].get(
+            "maya", {}).get(
+            "RenderSettings", {}).get(
+            "enable_all_lights", {})
```

Comment:
Is there a reason why even the `enable_all_lights` get should default to `{}` even though the value should be a boolean. I'd expect something like this:
```suggestion
            "enable_all_lights", False)
```

Or just:
```suggestion
            "enable_all_lights")
```